addons/hr_skills/views/hr_views.xml

```diff
@@ -0,0 +1,265 @@
+                                <tree>
+                                    <field name="line_type_id"/>
+                                    <field name="name"/>
+                                    <field name="sequence" widget="handle"/>
+                                    <field name="description"/>
+                                    <field name="date_start"/>
+                                    <field name="date_end"/>
+                                </tree>
+                            </field>
+                        </div>
+                        <div class="col-lg-4 o_group_skills">
+                            <separator string="Skills"/>
+                            <field nolabel="1" name="employee_skill_ids"  widget="hr_skills">
+                                <tree>
+                                    <field name="skill_type_id" invisible="1"/>
+                                    <field name="skill_id"/>
+                                    <field name="skill_level_id"/>
+                                    <field name="progress" widget="progressbar"/>
+                                </tree>
+                            </field>
+                        </div>
+                    </div>
+                </page>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="action_hr_resume_type" model="ir.actions.act_window">
+        <field name="name">Resumé Line Types</field>
+        <field name="res_model">hr.resume.line.type</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+
+    <menuitem
+            id="menu_human_resources_configuration_resume"
+            name="Resumé"
+            parent="hr.menu_human_resources_configuration"
+            sequence="4"
+            groups="base.group_no_one"/>
+
+    <menuitem
+        id="hr_menu_resume_line_type"
+        name="Types"
+        action="action_hr_resume_type"
+        parent="hr_skills.menu_human_resources_configuration_resume"
+        sequence="3"
+        groups="base.group_no_one"/>
+
+    <!-- Skills -->
+
+    <record id="view_employee_skill_level_tree" model="ir.ui.view">
+        <field name="name">hr.skill.level.tree</field>
+        <field name="model">hr.skill.level</field>
+        <field name="arch" type="xml">
+            <tree string="Skill Levels">
+                <field name="sequence" widget="handle"/>
+                <field name="name"/>
+                <field name="progress" widget="progressbar"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="view_employee_skill_tree" model="ir.ui.view">
+        <field name="name">hr.skill.tree</field>
+        <field name="model">hr.skill</field>
+        <field name="arch" type="xml">
+            <tree string="Skill Levels">
+                <field name="name"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="view_employee_skill_level_form" model="ir.ui.view">
```

Comment:
hr_skill_level_view_form according to guidelines.

You can check and apply the naming guidelines to all the views/actions/menuitems.